xblockutils/resources.py

```diff
@@ -85,3 +88,14 @@ def load_scenarios_from_path(self, relative_scenario_dir, include_identifier=Fal
                     scenarios.append((identifier, title, scenario))
 
         return scenarios
+
+class MakoResourceLoader(ResourceLoader):
```

Comment:
This doesn't really feel like a separate class to me.  Django and Mako templates aren't mutually exclusive in any way, so how about just adding a new method `render_mako_template()` to `ResourceLoader`?  (We could also deprecate the method name `render_template()` and call it `render_django_template()` instead for consistency, if you like that better.)